rc/filetype/sh.kak

```diff
@@ -16,6 +16,15 @@ provide-module sh %[
 
 add-highlighter shared/sh regions
 add-highlighter shared/sh/code default-region group
+
+# Languages
+try %{
+require-module python
+add-highlighter shared/sh/python region 'python.*?-c *"' '[^\\]"' regions
```

Comment:
the `python.*-c` could potentially yield an undesired match, maybe `python[^\n]* -c"` instead?

Reply:
That does not really solve the problem of undesired matches, only limits it. 
It also limits the normal usage, i.e. you can no longer do
```
python \
-c "
...
"
```

Reply:
-c flag also accepts string in single quotes, so maybe use `['"]` here too?